libraries/helpers.rb

```diff
@@ -256,7 +256,7 @@ def platform_supports_dcv?
 def aws_domain
   # Set the aws domain name
   aws_domain = "amazonaws.com"
-  aws_domain = "#{aws_domain}.cn" if node['cluster']['region'].start_with?("cn-")
```

Comment:
Why don't we set the region for tests instead of changing the code?
The region should always be set, correct?

Reply:
Yes, but this was always a pain point when I had to test on an instance not in a cluster: example run instance from a ParallelCluster AMI.

Reply:
I understand but I'm always reluctant to "weaken" production code for testing reasons, unless it is unavoidable.